vhd/packer/windows-vhd-builder.json

```diff
@@ -23,12 +28,12 @@
             "location": "{{user `location`}}",
             "vm_size": "{{user `vm_size`}}",
             "os_type": "Windows",
-            "image_publisher": "MicrosoftWindowsServer",
-            "image_offer": "WindowsServer",
-            "image_sku": "2019-Datacenter-Core-smalldisk",
-            "image_version": "17763.1339.2007101755",
+            "image_publisher": "{{user `image_publisher`}}",
+            "image_offer": "{{user `image_offer`}}",
+            "image_sku": "{{user `image_sku`}}",
+            "image_version": "{{user `image_version`}}",
             "resource_group_name": "{{user `resource_group_name`}}",
-            "capture_container_name": "aksengine-vhds-windows-ws2019",

```

Comment:
this will change from `ws2019` to `2019` by default, is that okay?

Reply:
I think it should be ok. This is only used in the paths in the storage account the VHD gets captured into correct?

Reply:
yeah should be fine, we get the URI directly from the output so we don't expect it to be in a specific place 👍